src/app/visual/checkbox/checkbox-visual.component.html

```diff
@@ -66,3 +66,72 @@
     [checked]="false">
   </sky-checkbox>
 </div>
+<div>
+  <h3>
+    Template-driven checkbox
+  </h3>
+  <p>
+    <sky-checkbox
+      ngModel
+      [required]="required"
+      #fooModel="ngModel"
+    >
+      <sky-checkbox-label
+        [ngClass]="{'sky-control-label-required' : required}"
+      >
+        Check me
+      </sky-checkbox-label>
+    </sky-checkbox>
+  </p>
+
+  <table>
+    <tr>
+      <th>Touched</th>
+      <td>{{ fooModel.touched }}</td>
+    </tr>
+    <tr>
+      <th>Pristine</th>
+      <td>{{ fooModel.pristine }}</td>
+    </tr>
+    <tr>
+      <th>Valid</th>
+      <td>{{ fooModel.valid }}</td>
+    </tr>
+  </table>
+
+  <h3>
+    Reactive checkbox
+  </h3>
+
+  <form
+    [formGroup]="reactiveFormGroup"
+    novalidate
+  >
+      <sky-checkbox
+        formControlName="reactiveCheckbox"
+      >
+        <sky-checkbox-label
+          [ngClass]="{'sky-control-label-required' : required}"
```

Comment:
I considered that, but our docs instruct consumers to add it themselves.

https://developer.blackbaud.com/skyux/components/form#required-fields

I could go either way on this since we don't have a precedent for it yet. Is there ever a scenario where we *wouldn't* want the required class to be automatically assigned to a required form element?

Reply:
I would lean toward enforcing a prescribed style, instead of relying on consumers to read our documentation. We should probably add this CSS class to our other form controls to make them all consistent as well.

Reply:
Done.